src/filters/dec_scte35.c

```diff
@@ -550,6 +578,7 @@ static const GF_FilterCapability SCTE35DecCaps[] =
 #define OFFS(_n)	#_n, offsetof(SCTE35DecCtx, _n)
 static const GF_FilterArgs SCTE35DecArgs[] =
 {
+	{ OFFS(pass), "pass-through mode, sets 'CueStart' at each splice point", GF_PROP_BOOL, "false", NULL, 0},
```

Comment:
I would rather call it 'mode' and explicit that one mode is 23001-18 and the other is video passthru adding the 'sctb' property at event time.

Opinion @jeanlf ?

Reply:
like `mode=23001-18` and `mode=passthru`?